CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Change Log
 
 All notable changes to this project will be documented in this file using [Semantic Versioning](http://semver.org/).
 
+## [0.4.8] - 2020-01-20
+### Fixed
```

Comment:
@tanjabrzak please create an [issue](https://github.com/lumoslabs/aleph/issues) with a description the problem and add the link here like [this](https://github.com/lumoslabs/aleph/blob/fa4d01cc37ae9da01eed0d5f8cfc1b60b1d18c2c/CHANGELOG.md#fixed-3)